app/preview/views.py

```diff
@@ -0,0 +1,46 @@
+import os
+import tempfile
+
+from flask import Blueprint, request, jsonify
+
+from flask.helpers import make_response, send_from_directory
+from app.doc2html.views import process_html_path
+
+htmlByBLobBlueprint = Blueprint('doc2htmlByBLob', __name__)
+
+
+@htmlByBLobBlueprint.route("/doc2htmlByBLob", methods=["POST"])
+def doc_2_html_from_bLob():
+    file = request.files.get('doc')
+    resp = build_response(file)
+    resp.headers.add("Access-Control-Allow-Origin", "*")
```

Comment:
CORS should be somewhere generic. not inside specific module (blueprint). Try to google up some best practice for flask and CORS

Reply:
use flask-cors